packages/eslint-config-eslint/nodejs.js

```diff
@@ -1,14 +1,24 @@
 "use strict";
 
-const recommendedScriptConfig = require("eslint-plugin-n/configs/recommended-script");
-const recommendedModuleConfig = require("eslint-plugin-n/configs/recommended-module");
+const {
+    configs: {
+        "flat/recommended-script": recommendedScriptConfig,
+        "flat/recommended-module": recommendedModuleConfig
+    }
+} = require("eslint-plugin-n");
```

Comment:
[**"eslint-plugin-n/configs/recommended-script"**](https://github.com/eslint-community/eslint-plugin-n/blob/master/configs/recommended-script.js#L4) and [**"eslint-plugin-n/configs/recommended-module"**](https://github.com/eslint-community/eslint-plugin-n/blob/master/configs/recommended-script.js#L4) are deprecated.